firmware/src/usb_descriptors.c

```diff
@@ -98,13 +101,23 @@ static const uint8_t g_config_descriptor[] = {
   // Configuration Descriptor Header
   0x09,  // Size of this descriptor
   USB_DESCRIPTOR_CONFIGURATION,  // Descriptor type
-  0x6c, 0x00,  // Total length of data for this cfg
+  0x74, 0x00,  // Total length of data for this cfg
   4,  // Number of interfaces in this cfg
   1,  // Index value of this configuration
   0,  // Configuration string index
   USB_ATTRIBUTE_DEFAULT | USB_ATTRIBUTE_SELF_POWERED,  // Attributes
   USB_POWER_CONSUMPTION,  // Max power consumption
 
+  // Interface Association Descriptor: CDC Function 1
+  0x08,   // Size of this descriptor in bytes
+  0x0B,   // Interface association descriptor type
+  0,   // The first associated interface
+  0x02,   // Number of contiguous associated interface
```

Comment:
Likewise you're a line off here. I think it should be "Number of contiguous associated interfaces", because its the number of associated interfaces which are contiguous, assuming I'm interpreting the comment correctly.